recipes/phylogenize/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.91" %}
+{% set sha256 = "3d428440f7a8aed3be18bbe094d3afe94f2d5357daa715d13e963c665384f2ab" %}
+{% set name = "phylogenize" %}
+
+package:
+  name: phylogenize
+  version: "{{ version }}"
+
+source: 
+  url: https://github.com/pbradleylab/phylogenize/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  host:
+    - perl 5.32.1
+    - r-base 4.3.1
```

Comment:
It should be a pinning expression, e.g `r-base >=4.3.1`, see a full example https://github.com/bioconda/bioconda-recipes/blob/master/recipes/bioconductor-a4classif/meta.yaml

Reply:
Should be good now